_docs/guides/integrating-vms.md

```diff
@@ -15,11 +15,16 @@ this infrastructure as a single mesh.
 
 > Note: this guide is still under development and only tested on Google Cloud Platform.
   On IBM Bluemix or other platforms where overlay network of Pods is isolated from VM network,
-  VMs cannot initiate any direct communication to Kubernetes Pods when using Istio.
+  VMs cannot initiate any direct communication to Kubernetes Pods even when using Istio.
 
 ## Overview
 
-TBD
+
+<figure><img src="./img/mesh-expansion.svg" alt="BookInfo Application with Istio Mesh Expansion" title="BookInfo Application with Istio Mesh Expansion" />
+<figcaption>BookInfo Application with Istio Mesh Expansion</figcaption></figure>
+
+<!-- source of the drawing https://docs.google.com/drawings/d/1gQp1OTusiccd-JUOHktQ9RFZaqREoQbwl2Vb-P3XlRQ/edit -->
```

Comment:
Can you ensure that this is in the shared folder ?

Reply:
done: "Mesh Expansion / Bookinfo" has been moved from My Drive to Presentations.